packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/proposeConfig.ts

```diff
@@ -42,7 +42,7 @@ const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInpu
 
   return {
     f: aggregator.config.f,
-    proposalId: flags.proposalId,
+    proposalId: flags.configProposal,
```

Comment:
Let's add support for both and add a comment that this is an ad-hoc alias as requested by eng ops.